pkg/operator/staticpod/file_utils_linux.go

```diff
@@ -0,0 +1,33 @@
+//go:build linux
```

Comment:
I think we should provide an implementation for operating systems other than linux, especially for people who run unit tests on their local machines, wdyt?

Reply:
Well, the syscall doesn't exist on other systems. We would need to mock it and then somebody could accidentally use it without knowing there is no promise of atomicity.

Reply:
Do you mean it doesn't have to be atomic? Because it can't be 🙂